### DISCUSSION TITLE
Maven versioning priority

### DISCUSSION THREAD

**Discussion by ajax-kovtunov-o** [renovatebot/renovate#29056] [Request Help]
### What would you like help with?

None

### How are you running Renovate?

None

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

github action

### Please tell us more about your question or problem

I'm so appreciate about support of renovate team and wood like to ask again.

Configuration
```json
"packageRules": [
    {
      "matchDatasources": ["maven", "gradle"],
      "managers": [
        "gradle"
      ],
      "registryUrls": [
        "s3://myorg-maven-repository/releases",
        "s3://myorg-maven-repository/snapshots"
      ],
      "automerge": false,
      "allowedVersions": "/(RELEASE|MASTER-SNAPSHOT)/"
    }
  ]
```

So, I want to setup configuration that will be prefer `MASTER-SNAPSHOT` versions over of `RELEASE`. In most cases we want to update to `MASTER-SNAPSHOT`.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Duplicate of https://github.com/renovatebot/renovate/discussions/29031

If you @ mention us to get attention, as well as post duplicates, you are less unlikely to get help.

**Comment by ajax-kovtunov-o** [comment]
The '@' isn't used to get attention, just to say thank you for the previous
help. I don't know about the rule to not use mentions. I'm sorry.
